sky/packages/sky/lib/src/gestures/arena.dart

```diff
@@ -45,6 +45,7 @@ class GestureArenaEntry {
 class _GestureArenaState {
   final List<GestureArenaMember> members = new List<GestureArenaMember>();
   bool isOpen = true;
+  bool isHeld = false;
```

Comment:
I'm assuming closing the arena automatically releases it; am I wrong?

If this is a lifecycle and you can't go backwards, then:
```dart
enum ArenaLifecycle { open, held, closed }
```